mailbox/api/src/test/java/org/apache/james/mailbox/events/GroupContract.java

```diff
@@ -73,6 +77,23 @@ default void dispatchShouldNotThrowWhenAGroupListenerFails() {
             .doesNotThrowAnyException();
     }
 
+    @Test
+    default void failingGroupListenersShouldNotAbortGroupDelivery() {
```

Comment:
This test method addition deserves to be in a separate commit explaining it adds some excpetion handling.

This is a separate commit in the POC, so should it be here

See https://github.com/linagora/james-project/pull/2070/commits/cbe4dc5241cdfeb9e2a2bbc53a9a2196604881fd